test/test_torch.py

```diff
@@ -6265,14 +6265,32 @@ def load_bytes():
             return data
 
         fileobject_lambdas = [lambda: test_file_path, load_bytes]
-        map_locations = [map_location, {'cuda:0': 'cpu'}, 'cpu']
+        map_locations = [map_location, {'cuda:0': 'cpu'}, 'cpu', torch.device('cpu')]
 
         for fileobject_lambda in fileobject_lambdas:
             for map_location in map_locations:
                 tensor = torch.load(fileobject_lambda(), map_location=map_location)
                 self.assertIsInstance(tensor, torch.FloatTensor)
                 self.assertEqual(tensor, torch.FloatTensor([[1.0, 2.0], [3.0, 4.0]]))
 
+    @unittest.skipIf(not torch.cuda.is_available(), 'no CUDA')
+    def test_serialization_map_location_cuda(self):
```

Comment:
I can't see why do we need this extra test. It should be very easy to just add the extra cases to the one above, and change `assertIsInstance(tensor, torch.FloatTensor)` to something more robust

Reply:
Can we also get tests for more strings? `cuda`, `cuda:0`, `'cuda:{}'.format(torch.cuda.device_count() - 1)`, etc.

Reply:
I had it as a separate test because I wanted to ensure that it got skipped when CUDA was not available. Merging them is a bit cleaner and I can just use torch.cuda.is_available() inside the test. I added the extra strings you recommended.